lib/portabilling/account_session_manager.ex

```diff
@@ -170,9 +170,14 @@ defmodule Portabilling.AccountSessionManager do
   end
 
   @new_password_chars Enum.concat([?a..?z, ?A..?Z, ?0..?9])
+  @default_account_password_length 12
```

Comment:
Moving this contact nearer to the place where this environment variable is read and set will be better.
Similar to for example the `OTP_IGNORE_ACCOUNTS` env variable, by checking for `nil` in `case` and assigning default value.